test/e2e_node/device_plugin_test.go

```diff
@@ -103,6 +105,18 @@ func testDevicePlugin(f *framework.Framework, pluginSockDir string) {
 				return nodes == 1
 			}, time.Minute, time.Second).Should(gomega.BeTrue())
 
+			v1alphaPodResources, err = getV1alpha1NodeDevices()
+			framework.ExpectNoError(err)
+
+			v1PodResources, err = getV1NodeDevices()
+			framework.ExpectNoError(err)
+
+			// Before we run the device plugin test, we need to ensure
+			// that the cluster is in a clean state and there are no
+			// pods running on this node.
+			framework.ExpectEqual(len(v1alphaPodResources.PodResources), 0)
```

Comment:
```suggestion
			gomega.Expect(v1alphaPodResources.PodResources).To(gomega.BeEmpty(), "should have no pod resources")
```

If this goes wrong, it will print the unexpected pod resources.

Reply:
Fixed, Thanks.